### PR TITLE
Add Laravel 8 support

### PR DESCRIPTION
**Pull request by SammyK**
### Description

This PR adds Laravel 8 to the CI tests. No changes were required for the Laravel integration.

### Reviewer checklist
- [x] Appropriate labels assigned.
- [x] Milestone is set.
- [x] Changelog has been added to the release document. For community contributors the reviewer is in charge of this task.

### CODE REVIEW

File: tests/Frameworks/Laravel/Version_8_x/routes/web.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Route::get('simple', [CommonSpecsController::class, 'simple'])->name('simple_route');
+Route::get('simple_view', [CommonSpecsController::class, 'simple_view']);
+Route::get('error', [CommonSpecsController::class, 'error']);
+Route::get('eloquent/get', [EloquentTestController::class, 'get']);
+Route::get('eloquent/insert', [EloquentTestController::class, 'insert']);
+Route::get('eloquent/update', [EloquentTestController::class, 'update']);
+Route::get('eloquent/delete', [EloquentTestController::class, 'delete']);
+Route::get('eloquent/destroy', [EloquentTestController::class, 'destroy']);
+Route::get('eloquent/refresh', [EloquentTestController::class, 'refresh']);
```
Comment (morrisonlevi) [inline tests/Frameworks/Laravel/Version_8_x/routes/web.php:26]: Does this support closures?
What about `[new CommonSpecsController(), 'simple']`?

Comment (SammyK) [reply tests/Frameworks/Laravel/Version_8_x/routes/web.php:26]: Yes, it [supports closures](https://laravel.com/docs/8.x/routing#basic-routing). But when I tried using an instance of the controller I got this error:

```
production.ERROR: Object of class App\Http\Controllers\CommonSpecsController could not be converted to string
```

**Review by labbati — APPROVED** [review]
Great to have this tested. Thank you Sammy.

File: tests/Frameworks/Laravel/Version_8_x/routes/web.php (hunk 1 above)
Comment (morrisonlevi) [reply tests/Frameworks/Laravel/Version_8_x/routes/web.php:26]: So, it doesn't support `callable` generally; just specific forms of callables. I suppose we should add tests for the other forms. I'm not aware of a JIRA for this; are you?

Comment (SammyK) [reply tests/Frameworks/Laravel/Version_8_x/routes/web.php:26]: There wasn't one so I created one. :)

**Pull request merged**